modules/kuberay-cluster/main.tf

```diff
@@ -73,3 +73,100 @@ data "kubernetes_service" "head-svc" {
   }
   depends_on = [helm_release.ray-cluster]
 }
+
+# Allow ingress to the kuberay head from outside the cluster
+resource "kubernetes_network_policy" "kuberay-head-network-policy" {
+  count = var.disable_network_policy ? 0 : 1
+  metadata {
+    name      = "terraform-kuberay-head-network-policy"
+    namespace = var.namespace
+  }
+
+  spec {
+    pod_selector {
+      match_labels = {
+        "ray.io/is-ray-node" : "yes"
+      }
+    }
+
+    ingress {
+      # Ray Client server
+      ports {
+        port     = "10001"
```

Comment:
Can we get the port from other resource? since we already import the ray cluster above

Reply:
Is there a way to pull these values from the helm_release via outputs? I don't think I have access to these ports unless I set them as variables on the module and push them through to the helm chart.